balancer_switching_test.go

```diff
@@ -79,6 +80,13 @@ func startServers(t *testing.T, numServers int, maxStreams uint32) ([]*server, f
 	}
 }
 
+func errorDesc(err error) string {
```

Comment:
Where is this called?

Reply:
Currently called from `balancer_switching_test.go`, `server_test.go` and `pickfirst_test.go`. All references to this from the last file are removed in this PR.